src/action/github.ts

```diff
@@ -1,10 +1,8 @@
+import * as core from '@actions/core';
 import {Action} from '../action';
 import {Output} from '../config/output';
 import {Logger} from '../logging';
 
-/* eslint-disable-next-line import/no-commonjs, @typescript-eslint/no-var-requires */
```

Comment:
I am pretty sure that I had to do this.
But since tests are passing, I think its okay now.

Maybe that was on my macos machine when I compiled that locally. Dunno...